sys/riscv/include/cherireg.h

```diff
@@ -193,17 +191,12 @@
 #define	CHERI_EXCCODE_STORE_LOCALCAP	0x16
 #define	CHERI_EXCCODE_PERM_SEAL		0x17
 #define	CHERI_EXCCODE_SYSTEM_REGS	0x18
-#define	CHERI_EXCCODE_PERM_CCALL	0x19
-#define	CHERI_EXCCODE_CCALL_IDC		0x1a
+#define	CHERI_EXCCODE_PERM_CINVOKE	0x19
+#define	CHERI_EXCCODE_CINVOKE_IDC	0x1a
```

Comment:
This should be deprecated and removed. It only existed whilst CCallFast had a delay slot in CHERI-MIPS, raised when the delay slot tried to access IDC, but (a) the delay slot later went away (b) CHERI-MIPS itself went away. We should update cheri-specification accordingly too.

Reply:
I've added the ISA change to the de-MIPS PR.